qcodes_contrib_drivers/drivers/Spectrum/M4i.py

```diff
@@ -32,7 +32,8 @@
         log.info('M4i: adding header_dir %s to sys.path' % header_dir)
         sys.path.append(header_dir)
     import pyspcm
-except (ImportError, OSError) as ex:
+    import py_header.spcerr
```

Comment:
Is there any specific reason to import this here?

Reply:
spcerr is needed and it is in the directory added to the path on line 33. So, yes, the import is needed.